data-es2016plus.js

```diff
@@ -2142,6 +2142,57 @@ exports.tests = [
       duktape2_0: false,
     },
   },
+  {
+    name: 'object rest/spread properties',
+    significance: 'large',
```

Comment:
For reviewers: notice that I changed significance to large because of huge amount of code on GitHub which uses this feature more than 3 years already

Reply:
I'm not sure that this change makes sense. At this moment, "large" significance only for features with new conceptions like async functions, zones, realms, etc.

Reply:
Ok, reverted